src/amp-story-player.js

```diff
@@ -98,6 +109,17 @@ export class AmpStoryPlayer {
 
     /** @private {number} */
     this.currentIdx_ = 0;
+
+    /** @private {?SwipingState} */
+    this.swipingState_ = null;
```

Comment:
Nit: just to have a complete state machine, `NOT_SWIPING`? We should also set it back to this state on `touchend`

Reply:
Done.